ExpandingMenu/Classes/ExpandingMenuButton.swift

```diff
@@ -7,56 +7,63 @@
 
 import UIKit
 import AudioToolbox
+import AVFoundation
 
-public struct CustomAnimationOptions : OptionSet {
+public struct CustomAnimationOptions: OptionSet {
     public let rawValue: Int
     public init(rawValue: Int) { self.rawValue = rawValue }
-    
-    public static let MenuItemRotation = CustomAnimationOptions(rawValue: 1)
-    public static let MenuItemBound = CustomAnimationOptions(rawValue: 2)
-    public static let MenuItemMoving = CustomAnimationOptions(rawValue: 4)
-    public static let MenuItemFade = CustomAnimationOptions(rawValue: 8)
-    
-    public static let MenuButtonRotation = CustomAnimationOptions(rawValue: 16)
-    
-    public static let Default: CustomAnimationOptions = [MenuItemRotation, MenuItemBound, MenuItemMoving, MenuButtonRotation]
-    public static let All: CustomAnimationOptions = [MenuItemRotation, MenuItemBound, MenuItemMoving, MenuItemFade, MenuButtonRotation]
+    public static let rotate = CustomAnimationOptions(rawValue: 1 << 0)
+    public static let bound = CustomAnimationOptions(rawValue: 1 << 1)
+    public static let move = CustomAnimationOptions(rawValue: 1 << 2)
+    public static let fade = CustomAnimationOptions(rawValue: 1 << 3)
+    public static let menuButtonRotate = CustomAnimationOptions(rawValue: 1 << 4)
+    public static let `default`: CustomAnimationOptions = [rotate, bound, move, menuButtonRotate]
+    public static let all: CustomAnimationOptions = [rotate, bound, move, fade, menuButtonRotate]
 }
 
 open class ExpandingMenuButton: UIView, UIGestureRecognizerDelegate {
     
-    public enum ExpandingDirection {
+    @objc public enum ExpandingDirection: Int {
         case top
         case bottom
     }
     
-    public enum MenuTitleDirection {
+    @objc public enum MenuTitleDirection: Int {
         case left
         case right
     }
     
+    @objc public enum HapticFeedbackStyle: Int {
+        case light
+        case medium
+        case heavy
+        case none
+    }
+    
     // MARK: Public Properties
-    @objc open var menuItemMargin: CGFloat = 16.0
+    @objc open var menuItemMargin: CGFloat = 7.0
+    @objc open var menuButtonHapticStyle: HapticFeedbackStyle = .medium
+    @objc open var menuItemsHapticStyle: HapticFeedbackStyle = .light
     
-    @objc open var allowSounds: Bool = true {
+    @objc open var playSound: Bool = false {
```

Comment:
Please use `true` or please describe the reason why this value should be `false`.